mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/service/DatentransferService.java

```diff
@@ -0,0 +1,38 @@
+package de.muenchen.mobidam.service;
+
+import de.muenchen.mobidam.domain.dtos.DatentransferDTO;
+import de.muenchen.mobidam.domain.mappers.DatentransferMapper;
+import de.muenchen.mobidam.repository.DatentransferRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@AllArgsConstructor
+@Service
+public class DatentransferService {
+
+    private static final int PAGE_SIZE = 10;
+
+    private final DatentransferRepository datentransferRepository;
+    private final DatentransferMapper datentransferMapper;
+
+    public Optional<Iterable<DatentransferDTO>> getBySchnittstelle(String schnittstelleId, int page) {
+        List<DatentransferDTO> dtos = new ArrayList<>();
+        Pageable pageable = PageRequest.of(page, PAGE_SIZE);
+        UUID schnittstelleUUID;
+        try {
+            schnittstelleUUID = UUID.fromString(schnittstelleId);
+        } catch (IllegalArgumentException ex) {
+            return Optional.empty();
```

Comment:
Das ist immer noch so ein WFT-Moment. Du gibst eine leere Menge zurück (was prinzipiell valide wäre) und verwandelst die dann in einen Fehler.
Besser wäre es, Du wirfst die IllegalArgExc selbst zurück und ein globaler ExceptionMapper verwandelt das dann in BAD_REQUEST. Was meinst Du?